iot-e2e-tests/android/app/src/androidTest/java/com/microsoft/azure/sdk/iot/android/iothub/errorinjection/twin/ReportedPropertiesErrInjAndroidRunner.java

```diff
@@ -24,17 +21,14 @@
 import java.security.GeneralSecurityException;
 import java.util.Collection;
 
-@TestGroup26
+@TestGroup8
 @RunWith(Parameterized.class)
-public class ReportedPropertiesDeviceAndroidRunner extends ReportedPropertiesTests
+public class ReportedPropertiesErrInjAndroidRunner extends ReportedPropertiesErrInjTests
 {
     @Rule
     public Rerun count = new Rerun(3);
 
-    @Rule
-    public ReportHelper reportHelper = Factory.getReportHelper();
-
-    public ReportedPropertiesDeviceAndroidRunner(IotHubClientProtocol protocol, AuthenticationType authenticationType, ClientType clientType, String publicKeyCert, String privateKey, String x509Thumbprint)
+    public ReportedPropertiesErrInjAndroidRunner(IotHubClientProtocol protocol, AuthenticationType authenticationType, ClientType clientType, String publicKeyCert, String privateKey, String x509Thumbprint)
```

Comment:
Q - don't we have tests that take in the SSL context directly, or do they have a specific test class?

Reply:
We do have a single test for that in the SendMessages test suite

Reply:
For context, the SSLContext constructor was a very recent addition, and most of these tests were written before then.